deployment.yaml

```
@@ -559,6 +559,14 @@ objects:
         partitions: 1
       - topicName: ${KAFKA_HOST_INGRESS_P1_TOPIC}
         partitions: 1
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdJobInvocation
+  metadata:
+    name: duplicate-hosts-remover-job-name-${CJI_RANDOM_SUFFIX}
```

We ran into an issue on Friday where the job name was too long. Can we remove the `job-name` part and just use this?
```suggestion
    name: duplicate-hosts-remover-${CJI_RANDOM_SUFFIX}
```

---

Done!